doc/specs/collages.md

```diff
@@ -0,0 +1,140 @@
+# Collages
+
+## Table of contects
+
+1. Motivation
+1. Business rules
+1. Use cases
+
+## Motivation
+
+The collage system is an important feature.
+It provides an orthogonal way to either artist page or tagging system to organise Album, based (mostly) on Facts.
+
+## Business rules
+
+* Collage contains albums
```

Comment:
Why shouldn't they be able to contain other things? I think a collage of 'software with which you can convert FLAC to MP3' can be pretty valid.